tokio/src/stream/mod.rs

```diff
@@ -862,7 +862,7 @@ pub trait StreamExt: Stream {
     /// use std::time::Duration;
     /// # let int_stream = stream::iter(1..=3);
     ///
-    /// let mut int_stream = int_stream.timeout(Duration::from_secs(1));
+    /// let mut int_stream = Box::pin(int_stream.timeout(Duration::from_secs(1)));
```

Comment:
Wouldn't it be better to use `tokio::pin!` here?

Reply:
For doctests I debated a bit whether using `Box:: pin` or `tokio::pin!` would be better as far as avoiding distracting from the main point of the docs in question. Regardless though, in 0.3 it'll still be Unpin so this doctest change should simply be backed out